src/pages/ChatPage.js

```diff
@@ -1,44 +1,57 @@
-import React, { useState } from 'react';
-import './ChatPage.css';
+import React, { useState, useRef } from 'react';
+import styles from './ChatPage.module.css';
+import { Link } from 'react-router-dom';
+import profile from '../logo.svg';
 
 const ChatPage = ({location, history}) => {
     const name = location.state;
-    const [message, setMessage] = useState([]);
-    const userText=['안녕하세요', '샘플 데이터입니다', '반가워요', '날씨가 어때요?', '감사합니다', '좋은하루 되세요', '사랑해요ㅎ.ㅎ'];
-    const [value, setValue] = useState('');
-    const onChange = (event) => {
-        setValue(event.target.value);
+    const [messages, setMessages] = useState([]);
+    const userTexts=['안녕하세요', '샘플 데이터입니다', '반가워요', '날씨가 어때요?', '감사합니다', '좋은하루 되세요', '사랑해요ㅎ.ㅎ'];
+    const [typing, setTyping] = useState(false);
+    const [fullScreen, setFullScreen] = useState(false);
+    const inputRef = useRef();
```

Comment:
```suggestion
    const messageInputRef = useRef();
```

추후 input이 더 추가될 수 도 있으니 어떠한 input인지 조금 더 명확히 할 필요가 있습니다.